### ISSUE TITLE
Accuracy Vs F1-score

### ISSUE THREAD

**Issue by yassmine-lam** [chakki-works/seqeval#83]
Hi,

I am working on a sequence labeling problem. I am using seqeval to obtain accuracy and F1 scores. However, is it normal to get an accuracy significantly higher than F1-score (Acc 95.2% and average F-1 score : 52.4%)

I am very confused 

Thank u for ur help

**Comment by wzkariampuzha** [comment]
Yes, see here: https://towardsdatascience.com/accuracy-precision-recall-or-f1-331fb37c5cb9
